client/src/components/buttons/BlueLink.jsx

```diff
@@ -1,17 +1,17 @@
-import {Link} from 'react-router-dom'
+import { Link } from 'react-router-dom'
 import { AuthAction } from './styled'
 
-export default function Login({to, text}){
+export default function Button({to, text, children}){
     return (
         <AuthAction>
             {
                 to?
                 <Link to={to}>
-                    {text}
+                    {text||children}
                 </Link>
                 :
                 <button type='submit'>
-                    {text}
+                    {text||children}
```

Comment:
why not just stick with one prop, it will render it as text if text is passed and it will render the child element if that is passed?